### PR TITLE
remove outdated scan configuration

### LINKED ISSUE #1635: Section [scan] of pf.conf should be removed
Its outdated and was replaced (but never removed) by scan.conf

### PR DESCRIPTION
**Pull request by julsemaan**
# Description

Cleanup outdated [scan] section of pf.conf
# Impacts

Should be none
# Issue

fixes #1635 
# Delete branch after merge

(REQUIRED)
YES | NO

### CODE REVIEW

**Comment by fdurand** [comment]
Rebase please

File: lib/pf/web.pm [hunk 1]
```diff
@@ -201,10 +201,11 @@ sub generate_scan_start_page {
     my $logger = get_logger();
 
     $portalSession->stash({
-        timer           => $Config{'scan'}{'duration'},
+        # Hardcoded here since the scan section is gone.
+        # In case this codepath is still called (pf::web::release is still called by scan in some cases)
+        timer           => 60,
```
Comment (rouzier) [inline lib/pf/web.pm:206]: Could this be moved into the scanner configuration?

Comment (julsemaan) [reply lib/pf/web.pm:206]: Not even sure that codepath is still being used so I don't think we should really put efforts in it.

Would like to remove it completely but I'm not confident enough about it

**Comment by extrafu** [comment]
Rebase

**Comment by jrouzierinverse** [comment]
Bump

**Comment by fdurand** [comment]
I have rebased

**Pull request merged**